---
pagetitle: "TCGA Data Analysis with R"
title: "The Cancer Genome Atlas (TCGA) Data Analysis with R"
layout: full
editor: source
---
------------------------------------------------------------------------
üßë Instructor: [Md. Jubayer Hossain](https://hossainlab.github.io/)

üóì July 5, 2024 - July 6, 2024 \| 9:30 am - 12:30pm (Bangladesh Time)

üè® Medium - Zoom

üí• **Register** with [Google Forms](https://forms.gle/ke8U9uYxXmtvsgtk7) 
<!-- **Registration**: *Closed* -->

üí• **Registration Fee**: 1530BDT  (for students), 2040BDT (for professionals)

üìù To join private Telegram group for the course, follow instructions in the email you received after registration.
------------------------------------------------------------------------



# Welcome!
The TCGA Data Analysis with R course is designed to equip participants with the knowledge and skills necessary to effectively analyze and interpret data from The Cancer Genome Atlas (TCGA) using the R programming language. TCGA is a valuable resource for cancer researchers, providing comprehensive genomic and clinical data on various cancer types. This course will cover essential concepts, tools, and techniques for data preprocessing, exploratory data analysis, differential gene expression analysis, survival analysis, and data visualization using R. Participants will gain hands-on experience by working with real TCGA datasets and will learn to derive meaningful insights from complex cancer genomics data.

## Learning Objectives

-   Introduce you to the TCGA (The Cancer Genome Atlas) data available at the NCI's Genomic Data Commons (GDC).
-   Demonstrate how to access and import TCGA data using the R/Bioconductor package TCGAbiolinks (Colaprico et al. 2015).
-   Provide instruction to visualize copy number alteration and mutation data using the R/Bioconductor package maftools.

## Pre-requisites

-   Basic knowledge of R syntax
-   Understand the pipe operator ("%\>%") (help material <https://r4ds.had.co.nz/pipes.html>)
-   Understand the SummarizedExperiment data structure (help material <http://bioconductor.org/packages/SummarizedExperiment/>)

## Course Format

The course will be delivered through a combination of lectures, hands-on practical sessions, and interactive discussions. Participants will have access to real TCGA datasets and will be guided through step-by-step analysis using R. Additionally, participants will be provided with relevant learning resources, including code examples and data repositories, to support their learning outside the course hours.

## What you'll learn?
### Case Study 1: Pan Cancer Downstream Analysis BRCA
In this case study, we will delve into the world of bioinformatics and cancer research by performing a downstream analysis of the BRCA (Breast Invasive Carcinoma) cancer dataset from the Pan Cancer Atlas. Our primary focus will be on utilizing R programming and basic concepts of bioinformatics and data analysis to gain insights into this specific cancer type.

### Case Study 2: Pan Cancer Downstream Analysis of LGG (Low-Grade Glioma)
In this case study, we will embark on a bioinformatics journey to analyze the LGG (Low-Grade Glioma) cancer dataset from the Pan Cancer Atlas. Our focus will be on utilizing R programming and basic bioinformatics concepts to gain valuable insights into the genetic and molecular characteristics of LGG.

### Case Study 3:  Integration of Methylation and Expression Data for Adrenocortical Carcinoma (ACC)
In this case study, we will dive into the field of bioinformatics and cancer research by exploring the integration of methylation and gene expression data for Adrenocortical Carcinoma (ACC). ACC is a rare and aggressive cancer that originates in the adrenal cortex. Our primary focus will be on utilizing R programming to integrate these two types of data and gain deeper insights into the molecular characteristics of ACC.

### Case Study 4: ELMER Pipeline for Integrative Analysis of DNA Methylation and Gene Expression in Kidney Renal Clear Cell Carcinoma (KIRC)

In this case study, we will delve into the world of bioinformatics and cancer research by exploring the ELMER (Enhancer Linking by Methylation/Expression Relationships) pipeline for integrative analysis of DNA methylation and gene expression data in Kidney Renal Clear Cell Carcinoma (KIRC). KIRC is the most common type of kidney cancer, and understanding the epigenetic regulation of gene expression in this cancer can provide valuable insights into its molecular mechanisms.





